proxy-node-acceptance-tests/features/support/env.rb

```diff
@@ -2,13 +2,21 @@
 require 'capybara-screenshot/cucumber'
 require 'selenium/webdriver'
 require 'webdrivers/chromedriver'
+require 'yaml'
+
+ENVIRONMENTS = YAML.load_file(File.join(__dir__, 'environments.yml'))
+
+def env(key)
+  test_env = ENV.fetch('TEST_ENV', 'local')
```

Comment:
I would initialise test_env separately outside this method and use it here as it's going to be fetched multiple times otherwise. Not a huge performance consideration but a more "cleaner" approach. Just like we initialise a bunch of vars below.

Reply:
You can achieve the same effect with `test_env ||= ENV.fetch('TEST_ENV', 'local')` - welcome to the wonderful world of Ruby, Phil.